config/permission.php

```diff
@@ -92,6 +92,13 @@
 
     'display_permission_in_exception' => false,
 
+    /*
+     * By default, wildcard permission usage is disabled.
+     * When set to true, wildcard permission handling is enabled.
```

Comment:
I'm not a fan of the word "usage" here.
Perhaps the two sentences can be combined? Or maybe "By default wildcard permission lookups are disabled." and skip the second line?